README.md

```diff
@@ -87,14 +87,35 @@ and write the result to a local directory.
 
 ## One-time CLI installation
 
-The quick answer is to just run
-`go install github.com/abcxyz/abc/cmd/abc@latest` .
+There are two ways to install:
 
-This only works if you have `go` installed (https://go.dev/doc/install) and have
-the Go binary directory in your `$PATH` (try `PATH=$PATH:~/go/bin`).
+1.  The most official way:
 
-This is the temporary installation process until we start formally releasing
-precompiled binaries.
+    - Go to https://github.com/abcxyz/abc/releases
+    - Pick the most recent release that isn't an `-alpha` or `-rc` or anything,
+      just `vX.Y.Z`
+    - Download the `.tar.gz` file that matches your OS and CPU:
+
+      - Linux: you probably want `linux_amd64`, unless you're doing something
+        weird and awesome, and in that case you don't need advice
+      - Mac: `darwin` means macOS. `arm64` means M1/M2 macs. So a recent Macbook
+        Pro would use `darwin_arm64`, while an older Intel Mac would use
+        `darwin_amd64`
+
+      Example `curl` command, please substitute the version number you're
+      downloading:
+
+          $ curl -O -L https://github.com/abcxyz/abc/releases/download/v1.2.3/abc_1.2.3_linux_amd64.tar.gz
+
+    - Unpack the tar file:
+
+          $ tar xvzf abc_1.2.3_linux_amd64.tar.gz
```

Comment:
This can be streamlined so it doesn't leave the tar file laying around and only extracts the binary file:

```shell
  curl -sSL https://github.com/abcxyz/abc/releases/download/v1.2.3/abc_1.2.3_linux_amd64.tar.gz | tar -xzv abc
```